endpoints/purchase_orders.md

```diff
@@ -262,6 +263,205 @@ curl --location --request POST 'https://api.gearbox.com.au/public/v1/purchase_or
 }'
 ```
 
+## Update a purchase order
+
+`PATCH /public/v1/purchase_orders/:id` updates the purchase order with the provided `:id`.
```

Comment:
Just one thing that I forgot to mention, should we mention that you can clear `purchase_order_item` attributes by passing an empty string `''`?